tendrl/commons/message.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+from dateutil import parser
+from inspect import getframeinfo
+from inspect import stack
+import json
+from tendrl.commons.utils.time_utils import now
+import uuid
+
+
+class Message(object):
+    # priority[0] is debug....priority[5] is critical
+    # Use priorities and publishers when call create message obj
+    priorities = type("priorities", (),
+                      {"DEBUG": "debug", "INFO": "info", "NOTICE": "notice",
+                       "WARNING": "warning", "ERROR": "error",
+                       "CRITICAL": "critical"})
```

Comment:
@brainfunked It is like enum, they can pass like
     Message(Message.priorities.INFO, Message.publishres.COMMOM, {"message": "some message"})

so   Message.priorities.INFO will give info to message parameter, it is useful for avoid spell mistake and only this function can use.